klippy/extras/gcode_macro.py

```diff
@@ -3,7 +3,7 @@
 # Copyright (C) 2018-2021  Kevin O'Connor <kevin@koconnor.net>
 #
 # This file may be distributed under the terms of the GNU GPLv3 license.
-import traceback, logging, ast, copy
+import traceback, logging, ast, copy, time, calendar
```

Comment:
```suggestion
import traceback, logging, ast, copy, time
```

Remove calendar import, because using `time.time()` is sufficient.